config/routes.rb

```diff
@@ -0,0 +1,9 @@
+Rails.application.routes.draw do
+  namespace :v1, defaults: { format: 'json' } do
+    get 'greetings', to: 'greetings#index'
+  end
+  get '*page', to: 'static#index', constraints: ->(req) do
+    !req.xhr? && req.format.html?
+  end
+  root "static#index"
```

Comment:
- [ ] Navigating to your root route shows empty web page and printed error in the console even after installing all dependencies and running rails db:see, kindly fix this
![Screenshot from 2022-08-27 21-07-35](https://user-images.githubusercontent.com/66097371/187046521-d2d4715e-7dab-4b2d-a43e-1faef8edf6ea.png)